### PR TITLE
[#44285] Update Toggl Documentation

### PR DESCRIPTION
**Pull request by ghost**
[#44285] Update Toggl Documentation

https://community.openproject.org/work_packages/44285

### CODE REVIEW

File: docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md [hunk 1]
```diff
@@ -6,19 +6,39 @@ description: Time tracking with Toggl
 keywords: time tracking, toggl integration
 ---
 
-# Time tracking with Toggl
+# Time tracking with Toggl Track
 
-To track time to a work package via a start and stop button, we offer an integration for the time recording software (or online timesheet software) Toggl (https://toggl.com/).
+To track time to a work package via a start and stop button, we offer an integration for the time recording software (or online timesheet software) Toggl Track ([https://toggl.com/track/](https://toggl.com/track/)).
```
Comment (wielinde) [inline docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md]: @adam-op I think it is worth it to explain/clarify that the spent time gets saved in Toggl Track and not in OpenProject.

File: docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md [hunk 2]
```diff
@@ -6,19 +6,39 @@ description: Time tracking with Toggl
...
+This integration can be added for example for Google Chrome as [Toggle Track extension](https://chrome.google.com/webstore/detail/toggl-button-productivity/oejgccbfbmkkpaidnkphaiaecficdnfn) and it also exists for Firefox.
 
-![Toggl-integration](Toggl-integration.png)
+## Configuration
 
-This integration can be added for example for Google Chrome as [Chrome Extension](https://chrome.google.com/webstore/detail/toggl-button-productivity/oejgccbfbmkkpaidnkphaiaecficdnfn).
+On the cogwheel of the toggle track extension you reach the **Settings**, from there chose **Integrations**.
 
-You can then transfer the recorded times from Toggl to OpenProject via the API if needed.
+For OpenProject Domains that are located on openproject.com you are able to chose **Openproject - openroject.com** from the list of default integrations.
```
Comment (wielinde) [inline docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md]:
```suggestion
For OpenProject Domains that are located on openproject.com, which is the case for most Enterprise Cloud instances, you are able to chose **Openproject - openroject.com** from the list of default integrations.
```

File: docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md [hunk 3]
```diff
@@ -6,19 +6,39 @@ description: Time tracking with Toggl
...
+
+For any other OpenProject Domains you need to exactly add the **FQDN** to the **Custom URLs for Integration** and chose **OpenProject**. For example: community.openproject.org (wildcards are not supported by Toggle Track at the moment of writing)
```
Comment (wielinde) [inline docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md]: I believe that the abbreviation FQDM is not a term that most people know. I learnt it just some weeks ago.

```suggestion
For any other OpenProject domains you need to exactly add the domain name (including the subdomain if present) to the **Custom URLs for Integration** and chose **OpenProject**. For example: community.openproject.org (wildcards are not supported by Toggle Track at the moment of writing)
```

File: docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md [hunk 4]
```diff
@@ -6,19 +6,39 @@ description: Time tracking with Toggl
...
+
+![toggl-configuration-integrations-openproject](toggl-configuration-integrations-openproject.png)
+
+
+
+## Tracking time from OpenProject to Toggle Track
```
Comment (wielinde) [inline docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md]:
```suggestion
## Tracking time for OpenProject work packages to Toggl Track
```

File: docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md [hunk 5]
```diff
@@ -6,19 +6,39 @@ description: Time tracking with Toggl
...
+
+If this add-in is installed and the Integration URL is configured, a **Start/Stop Timer button** is displayed on the work package details view, which can be used to record times from OpenProject into toggl:
```
Comment (wielinde) [inline docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md]:
```suggestion
If this add-in is installed and the _Integration URL_ is configured, a **Start/Stop Timer button** is displayed on the work package details view, which can be used to record times from OpenProject into Toggl:
```

File: docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md [hunk 6]
```diff
@@ -6,19 +6,39 @@ description: Time tracking with Toggl
...
+
+![toggl-button-openproject-workpackage-detail-view](toggl-button-openproject-workpackage-detail-view.png)
+
+
+
+## Using toggle tracked time in OpenProject
```
Comment (wielinde) [inline docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md]:
```suggestion
## Copying time entries from Toggl Track to OpenProject
```

File: docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md [hunk 7]
```diff
@@ -6,19 +6,39 @@ description: Time tracking with Toggl
...
+
+If you would like to use the times tracked at toggle in OpenProject, you are able to transfer the recorded times back to OpenProject via the API if needed.
```
Comment (wielinde) [inline docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md]:
```suggestion
If you would like to use the times tracked in Toggl Track in OpenProject and you are a programmer, you are able to transfer the recorded time entries back to OpenProject via the API.
```

File: docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md [hunk 8]
```diff
@@ -6,19 +6,39 @@ description: Time tracking with Toggl
...
 Details can be found in our [API documentation](../../../../api/).
+
+Alternatively you could also take a look at this [simple open source ruby script](https://github.com/wielinde/toggl_track_to_openproject) which is not officially supported by OpenProject GmbH.
```
Comment (wielinde) [inline docs/user-guide/time-and-costs/time-tracking/toggl-integration/README.md]:
```suggestion
Alternatively you could also take a look at this [simple open source ruby script](https://github.com/wielinde/toggl_track_to_openproject) which is not officially supported by OpenProject GmbH. It selects time entries from Toggl Track via the Toggl Track API, accumulates the entries by date and saves them in OpenProject via the OpenProject API. Please read the instructions carefully. If you don't understand the README, please don't use it.
```

**Comment by ghost** [comment]
hi @wielinde 

i changed the README.MD as you suggested, please double check if this is good now. i think it can be merged

**Review by wielinde — APPROVED** [review]

**Pull request merged**